### ISSUE TITLE
Gracefully handle `web/config.js` loading errors 

### ISSUE THREAD

**Issue by gzdunek** [gravitational/teleport#39335]
`web/configs.js` script is used to load a base config for the React web app.
Unfortunately, we don't handle its loading error, so if it occurs, we try to run the app anyway which causes errors on a deeper level.

This happens, for example, when the page is reloaded very quickly, so the rate limiter is activated. 
The request fails with an error `429: Too many requests`, but the user see this:
<img width="611" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/06e9c680-af4c-4f8f-acdb-a1b24f3934eb">

We should add `onerror` handler to `<script src="/web/config.js" ></script>` (https://stackoverflow.com/a/4163527).
The error message should be shown to the user.

**Comment by zmb3** [comment]
cc @jentfoo - maybe we should relax the rate limits on web/config.js as well

**Comment by jentfoo** [comment]
Handling the error with a delayed retry seems like a more robust solution.  That would cover not just rate limits, but intermittent network or other unexpected failures.

**Comment by zmb3** [comment]
Yes, we will improve the error handling (which is the focus of this issue).

What we learned is today that a single user that presses cmd+R too quickly can break their own UI. This means a group of people in an office behind the same SNAT IP is very likely to see similar issues. (I pointed this concern out in the original rate limiting work, too).

**Comment by jentfoo** [comment]
I would need to look into the load of the endpoint in more detail, but I noticed immediately it's only limiting unauthenticated requests.  Could we instead provide authentication to this request when logged in?
